doc/ref_cert/README.md

```diff
@@ -1,9 +1,136 @@
+mainly leveraging test jobs, the common test case execution, the test
+result database and the S3 protocol to publish the artifacts. It's worth
+mentioning that dumping all results and logs required for conformance is
+already in place in CIRV (see
+[cntt-latest-zip](https://build.opnfv.org/ci/job/cntt-latest-zip/)) and
+Functest daily jobs (see
+[functest-hunter-zip](https://build.opnfv.org/ci/job/functest-hunter-zip/3/console)).
+
+It should be noted that
+[Xtesting CI](https://galaxy.ansible.com/collivier/xtesting) supports both
+centralized and distributed deployment models as described before. It has
+deployed the full toolchain in one small virtual machine to verify ONAP Openlab
+via Functest.
+
+<a name="testing-integration-requirements"></a>
+## Test case integration requirements
+
+To reach all goals (verification, validation, compliance and conformance)
+expected by CNTT, all test cases must be delivered as
+[Docker containers](https://www.docker.com/) and meet the requirements to
+simplify the CI toolchain setups:
+- the common test case execution
+- the unified way to manage all the interactions with the CI/CD components and
+  with third-parties (e.g. dump all test case logs and results for
+  conformance)
+
+For their part, the Docker containers simply enforce that the test cases are
+delivered with all runtime dependencies. This prevents lots of manual
+operations when configuring the servers running the test cases and prevents
+conflicts between the test cases due to any dependencies.
+
+It's worth mentioning that current
+[test cases selected by CNTT](./chapter03.md)
+already leverages [Xtesting](https://xtesting.readthedocs.io/en/latest/)
+which is a simple framework to assemble sparse test cases and to accelerate the
+adoption of CI/CD best practices. By managing all the interactions with the
+CI/CD components (test scheduler, test results database, artifact repository),
+it allows the developer to work only on the test suites without diving into
+CI/CD integration. Even more, it brings the capability to run heterogeneous
+test cases in the same CI toolchains thanks to a few,
+[quickly achievable](https://www.sdxcentral.com/articles/news/opnfvs-6th-release-brings-testing-capabilities-that-orange-is-already-using/2018/05/),
+constraints.
 
+The Docker containers proposed by the test projects must also embed
+[the Xtesting Python package](https://pypi.org/project/xtesting/) and
+[the related test case execution description files](https://git.opnfv.org/functest-xtesting/tree/docker/testcases.yaml)
+as required by Xtesting.
+
+Here are the issues tracking the updates of the existing OPNFV test
+projects:
+- Bottlenecks: https://github.com/cntt-n/CNTT/issues/510
+- NFVBench: https://github.com/cntt-n/CNTT/issues/865
+- StorPerf: https://github.com/cntt-n/CNTT/issues/673
+- VSPERF: https://github.com/cntt-n/CNTT/issues/511
+- YardStick: https://github.com/cntt-n/CNTT/issues/509
+
```

Comment:
This could be moved to the **CNTT RI and RC toolchains** section

Reply:
next PR